<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chtdocviewer</class>
 <widget class="QDialog" name="chtdocviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chtdocviewer</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>106</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>76</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DANH SÁCH CÁC TÀI LIỆU</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nhập website:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Danh sách website:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
